synapse/storage/events.py

```diff
@@ -586,6 +586,7 @@ def get_events(ev_ids):
 
                 current_state = yield resolve_events_with_factory(
                     state_sets,
+                    event_map={},
```

Comment:
Should this be None to be a bit more consistent?

Reply:
plausibly, but it's replaced in https://github.com/matrix-org/synapse/pull/2864/commits/ebfe64e3d69d0047ee9902a05beaf0249f11e072 anyway

Reply:
Yup, didn't realise until after i had submitted the review